Dockerfile

```diff
@@ -21,17 +22,13 @@ RUN chown -R bf /bio-formats-build
 USER bf
 WORKDIR /bio-formats-build
 RUN git submodule update --init
-
-USER bf
-WORKDIR /bio-formats-build
 RUN mvn clean install -DskipSphinxTests
 
-USER bf
 WORKDIR /bio-formats-build/bioformats
 RUN ant clean jars tools test
 
 
 ENV TZ "Europe/London"
 
 WORKDIR /bio-formats-build/bioformats/components/test-suite
-ENTRYPOINT ["/usr/bin/ant", "test-automated", "-Dtestng.directory=/data", "-Dtestng.configDirectory=/config"]
+ENTRYPOINT ["/usr/bin/ant", "test-automated", "-Dtestng.directory=/opt/data", "-Dtestng.configDirectory=/opt/config"]
```

Comment:
The other changes look fine, but this one I don't think has much point.  `/opt` is not, and has never been, a location for data storage; it's for optional third-party software packages, which this is not.

Reply:
On the mount point, rephrasing my comment in https://github.com/ome/bio-formats-build/pull/18#discussion_r179408315, my concern is primarily the divergence we created between the mount points of our Bio-Formats Docker images across different branches/repositories.

Below is the list of required changes that will need to happen in both situations:

1. use `/opt/xxx` as implemented initially, modify the image in this repository and the jobs from [east-ci](https://web-proxy.openmicroscopy.org/east-ci/)
1. use `/xxx` as proposed in this repository, modify https://github.com/openmicroscopy/bioformats/blob/v5.9.0/Dockerfile#L22, the jobs from [west-ci](https://web-proxy.openmicroscopy.org/east-ci/) and update the [devspace configuration](https://github.com/openmicroscopy/devspace/blob/v0.9.0/home/jobs/BIOFORMATS-test-folder/config.xml#L46)

Reverted in a7bf528  to limit the scope of this PR to the layer reduction.